ros_buildfarm/templates/status/js/setup.js

```diff
@@ -6,7 +6,8 @@ var QUERY_TRANSFORMS = {
   'GRAY': '<a class="i"></a>',
   'RED1': '<td><a class="m"></a>',
   'RED2': '</a><a class="m"></a><a',
-  'RED3': '<a class="m"></a></td>'
+  'RED3': '<a class="m"></a></td>',
+  'UNMAINTAINED' : '<span class="unmaintained"'
```

Comment:
If I understand the code correctly, it is not likely happens, but in theory if someone make `foo_unmaintaned_pkg` package, then it does not work correctly,